### PR TITLE
Release 3.4.0-rc2 (inc. backports)

### PR DESCRIPTION
**Pull request by mheon**
As the title says.

### CODE REVIEW

**Comment by Luap99** [comment]
@mheon Can you also backport https://github.com/containers/podman/pull/11614. This will fix issues on docs.podman.io and since readthedocs will use the actual tag we need to backport this as well.

File: version/version.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package version
-
+65;6402;1c
```
Comment (Luap99) [inline version/version.go:2]: Cat on the keyboard?

**Comment by mheon** [comment]
I'll throw the docs commit on top because it doesn't need to be part of the tag

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**